static/rapid.json

```diff
@@ -1,181 +1,55 @@
 {
  "releases": [
   {
-   "version": "1.24.17"
-  },
-  {
-   "version": "1.25.14"
+   "version": "1.28.2"
   },
   {
-   "version": "1.26.9"
+   "version": "1.27.6"
   },
   {
    "version": "1.27.6"
   },
   {
-   "version": "1.28.2"
+   "version": "1.27.6"
   },
   {
-   "version": "1.28.1"
+   "version": "1.27.5"
   },
   {
    "version": "1.27.4"
   },
   {
-   "version": "1.25.13"
-  },
-  {
-   "version": "1.26.8"
+   "version": "1.26.9"
   },
   {
-   "version": "1.27.5"
+   "version": "1.26.9"
```

Comment:
Can we remove the duplicates in code to prevent submitting 2 identical versions?